### PR TITLE
stars

### PR DESCRIPTION
**Pull request by rekhakilari**
- [DEMO LINK](https://rekhakilari.github.io/layout_stars/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,7 +7,79 @@
...
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+      </div>
```
Comment (VinnieJ-2k20) [inline src/index.html:22]: You don't need empty lines between single-line sibling elements.

```suggestion
      <div class="stars stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
```
Comment (VinnieJ-2k20) [inline src/style.css:39]: You can combine selectors that use same styles.
```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
  background-image: url(./images/star-active.svg);
}
```

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Solution looks good, but some code style needs final adjustments.

**Comment by rekhakilari** [comment]
- [DEMO LINK](https://rekhakilari.github.io/layout_stars/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_stars/report/html_report/)

**Comment by rekhakilari** [comment]
- [DEMO LINK](https://rekhakilari.github.io/layout_stars/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_stars/report/html_report/)

**Pull request closed without merging**